aws/data_source_aws_kms_alias_test.go

```diff
@@ -47,6 +48,19 @@ func testAccDataSourceAwsKmsAliasCheck(name string) resource.TestCheckFunc {
 			)
 		}
 
+		targetKeyArn := strings.Replace(
+			kmsKeyRs.Primary.Attributes["arn"],
+			kmsKeyRs.Primary.Attributes["name"],
+			fmt.Sprintf("key/%s", kmsKeyRs.Primary.Attributes["target_key_id"]),
+			1)
+		if attr["target_key_arn"] != targetKeyArn {
+			return fmt.Errorf(
+				"target_key_arn is %s; want %s",
+				attr["target_key_arn"],
+				targetKeyArn,
+			)
+		}
```

Comment:
How do you feel about `regexp` matching with `$` here? It seems that would be a little bit cleaner approach compared to modification of the original attribute.

Reply:
This would be awesome to have right now. I racked by brain all day wondering why my aws_efs_file_system instances were being recreated and discovered that it was due to a recent update to our stack that includes looking up a kms key: